### PR TITLE
Added recipe for Flask-Mail

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/flask-mail/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/flask-mail/meta.yaml]: Can we drop the dependencies below from the build?

File: recipes/flask-mail/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{%set name = "flask-mail" %}
+{%set camelName = "Flask-Mail" %}
```
Comment (jakirkham) [inline recipes/flask-mail/meta.yaml]: Could just make this name.

Comment (jakirkham) [inline recipes/flask-mail/meta.yaml]: Then the package name and url would come verbatim from the example.

Comment (pmlandwehr) [inline recipes/flask-mail/meta.yaml]: My impression was that we generally wanted package names to be lower-case, not camel case or mixed case. If that's not so, then sure, I can change it.

File: recipes/flask-mail/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/flask-mail/meta.yaml]: Done.

File: recipes/flask-mail/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/flask-mail/meta.yaml]: Ah sorry that is a bit vague. Basically was meaning one could do something like what is done in PR ( https://github.com/conda-forge/staged-recipes/pull/1415 ). In any event, this is just a suggestion. The effect is nearly identical.

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

File: recipes/flask-mail/meta.yaml (hunk 2 above)
Comment (pmlandwehr) [inline recipes/flask-mail/meta.yaml]: Ooh, good idea, will patch it into my basic template

Comment (jakirkham) [inline recipes/flask-mail/meta.yaml]: We should really sit down with `conda skeleton pypi` and get that to behave better. Ideally we should just be able to churn out recipes with it and use them without (or with little) modification.

**Pull request merged**